anchore_engine/services/policy_engine/engine/vulns/providers.py

```diff
@@ -67,13 +64,17 @@
     merge_nvd_metadata_image_packages,
 )
 from anchore_engine.subsys import logger, metrics
-from anchore_engine.utils import rfc3339str_to_datetime, timer
+from anchore_engine.utils import timer
 
 from .dedup import get_image_vulnerabilities_deduper, transfer_vulnerability_timestamps
 from .scanners import GrypeScanner, LegacyScanner
 from .stores import ImageVulnerabilitiesStore, Status
 
 
+class InvalidFeed(Exception):
```

Comment:
(nit) Not necessary for now but these exceptions should be defined in a common place so others can import them without importing the entire provider module. We should also have a base class with common behaviour and extend it. Otheriwse there's little point in raising custom exception

Reply:
Agreed. One benefit to this right now is that I can distinguish it from a ValueError in the controller and raise a more specific BadRequest error